packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/omnibox/OmniboxControllerIntegrationTests.java

```diff
@@ -50,14 +57,38 @@ public void givenContextEntriesWhenAQueryIsPerformedThenCommandsAreReturned() {
         Map<String, Object> firstQueryVariables = Map.of("contextEntries", omniboxContextEntries, "query", "");
         var firstQueryResult = this.omniboxCommandsQueryRunner.run(firstQueryVariables);
         List<String> allCommandLabels = JsonPath.read(firstQueryResult, "$.data.viewer.omniboxCommands.edges[*].node.label");
-        int allCommandLabelsNumber = allCommandLabels.size();
+        assertThat(allCommandLabels).hasSize(1).anyMatch(label -> Objects.equals(label, "Search"));
 
-        String query = "yello";
+        String query = "Sea";
         Map<String, Object> secondQueryVariables = Map.of("contextEntries", omniboxContextEntries, "query", query);
         var secondQueryResult = this.omniboxCommandsQueryRunner.run(secondQueryVariables);
-        List<String> filteredCommandsLabels = JsonPath.read(secondQueryResult, "$.data.viewer.omniboxCommands.edges[*].node.label");
-        assertThat(filteredCommandsLabels.size()).isNotEqualTo(allCommandLabelsNumber);
-        assertThat(filteredCommandsLabels).allMatch(commandLabel -> commandLabel.contains(query));
+        List<String> seaFilteredCommandsLabels = JsonPath.read(secondQueryResult, "$.data.viewer.omniboxCommands.edges[*].node.label");
+        assertThat(seaFilteredCommandsLabels).hasSize(1).anyMatch(label -> Objects.equals(label, "Search"));
+
+        Map<String, Object> thirdQueryVariables = Map.of("contextEntries", omniboxContextEntries, "query", "yello");
+        var thirdQueryResult = this.omniboxCommandsQueryRunner.run(thirdQueryVariables);
+        List<String> yelloFilteredCommandsLabels = JsonPath.read(thirdQueryResult, "$.data.viewer.omniboxCommands.edges[*].node.label");
+        assertThat(yelloFilteredCommandsLabels).isEmpty();
```

Comment:
Somehow changes made to this test were causing a massive loss of code coverage. I'll fix that.